pycsw/core/util.py

```diff
@@ -33,11 +33,15 @@
 import time
 import datetime
 import logging
-import urllib2
+try:
+    from urllib2 import Request, urlopen
+except:
+    from urllib.request import Request, urlopen
 from shapely.wkt import loads
 from owslib.util import http_post
 from pycsw.core.etree import etree
 
+from six import text_type
```

Comment:
Yes the less dependencies the better. Even if OWSLib brings those demos in at least pycsw will not have to keep account of them. 

Sent from my iPhone

> On Sep 12, 2015, at 19:19, Elliott Sales de Andrade <notifications@github.com> wrote:
> 
> In pycsw/core/util.py:
> 
> >  from shapely.wkt import loads
> >  from owslib.util import http_post
> >  from pycsw.core.etree import etree
> >  
> > +from six import text_type
> Sure, but @tomkralidis was mentioning less dependencies...
> 
> —
> Reply to this email directly or view it on GitHub.
>